app/workers/list_sync/destroy_worker.rb

```diff
@@ -0,0 +1,22 @@
+module ListSync
+  class DestroyWorker
+    include Sidekiq::Worker
+    sidekiq_options retry: 3, queue: 'sync'
+
+    def perform(linked_account_id, media_type, media_id)
+      pending_logs = LibraryEntryLog.for_entry(library_entry).pending
```

Comment:
where is this `library_entry` variable defined? I only saw it in the `UpdateWorker`. The `for_entry` is only looking for `media_type` and `media_id` only anyways so maybe just pass those in direct and change the method to accept both those parameters?